### DISCUSSION TITLE
Weekly Report 16.05. - 23.05.

### DISCUSSION THREAD

**Discussion by Umbranecis** [SE-TINF22B6/Plapy#136] [Weekly reports]
Greetings,

This week, due to private matters and sickness we were not able to do a lot.
We worked a bit more on testing all our features and now have tests for adding and removing things from the music Queue.

Further more, we purchased a domain where our homepage can run and one where we can host the bots API. We already have a server running for the bot, so we can just add both features to this server.

In the next sprint, we plan to make sure the api is callable on this domain, so far it only runs locally.
However, since the server runs on Nicks Account, and Nick is not available due to personal matters, we are nit sure if we can actually implement this following week.

Further more, we hope to finally finish the website upcoming week.

Let us know if there are any more questions

Team Plapy

**Comment by nuo-li** [comment]
What are the three metrics your team plan to measure for Plapy?
